pandas/core/series.py

```diff
@@ -419,14 +419,30 @@ def dtypes(self):
     def ftype(self):
         """
         Return if the data is sparse|dense.
+
+        .. deprecated:: 0.25.0
+           Use :func:`dtype` instead.
         """
+        warnings.warn("ftype is deprecated and will "
+                      "be removed in a future version. "
+                      "Use dtype instead.",
+                      FutureWarning, stacklevel=2)
+
         return self._data.ftype
 
     @property
     def ftypes(self):
         """
         Return if the data is sparse|dense.
+
+        .. deprecated:: 0.25.0
+           Use :func:`dtypes` instead.
         """
+        warnings.warn("ftypes is deprecated and will "
+                      "be removed in a future version. "
+                      "Use dtypes instead.",
```

Comment:
```suggestion
                      "Use Series.dtype instead.",
```

(dtype and dtypes are equivalent for Series, but I think we should point people to use dtype)